l10n_it_fatturapa_in/tests/test_import_fatturapa_xml.py

```diff
@@ -541,6 +541,29 @@ def test_24_xml_import(self):
         self.assertEqual(invoice.e_invoice_amount_tax, 0.0)
         self.assertEqual(invoice.e_invoice_amount_total, 34.32)
 
+    def test_25_xml_import(self):
+        arrotondamenti_attivi_account_id = self.env['account.account'].\
+            search([('user_type_id', '=', self.env.ref(
+                'account.data_account_type_other_income').id)], limit=1).id
+        arrotondamenti_passivi_account_id = self.env['account.account'].\
+            search([('user_type_id', '=', self.env.ref(
+                'account.data_account_type_direct_costs').id)], limit=1).id
+        arrotondamenti_tax_id = self.env['account.tax'].search(
+            [('type_tax_use', '=', 'purchase'),
+             ('amount', '=', 0.0)], order='sequence', limit=1)
+        self.env.user.company_id.arrotondamenti_attivi_account_id = (
+            arrotondamenti_attivi_account_id)
+        self.env.user.company_id.arrotondamenti_passivi_account_id = (
+            arrotondamenti_passivi_account_id)
+        self.env.user.company_id.arrotondamenti_tax_id = (
+            arrotondamenti_tax_id)
```

Comment:
Le righe qua sopra sono uguali a quelle del metodo `test_24_xml_import`. Sarebbero da mettere in un unico metodo.
Inoltre, consiglio di fare rebase per includere le modifiche di https://github.com/OCA/l10n-italy/pull/1377 e magari mettere il metodo in `l10n_it_fatturapa_in/tests/fatturapa_common.py`

Reply:
@eLBati Ok! Lo farò stasera